RiotSharp.Test/CacheTest.cs

```diff
@@ -14,7 +14,7 @@ public class CacheTest
 
         [TestMethod]
         [TestCategory("Cache")]
-        public void AddGet_TimeSpan_ShouldAddToTheCache_Test()
+        public void AddGet_TimeSpan_ShouldAddToTheCache_ReturnTrue()
```

Comment:
```suggestion
        public void AddGet_TimeSpan_ShouldAddToTheCache_ReturnsTrue()
```

Reply:
Do you think we should change the name to AddGet_TimeSpanShouldAddToTheCache_ReturnsTrue

So we keep the MethodName_StateUnderTest_ExpectedBehavior() naming convention?

Thanks for reviewing my pull request 👍

Reply:
yes, I think the more standardization the better :+1: